src/queries/models/match.py

```diff
@@ -68,15 +68,15 @@ def match_video_path(self):
         return Video.objects.values_list('path', flat=True).get(id=video_id)
 
     @property
-    def match_video_start_time(self):
+    def match_video_time_span(self):
         """
         TODO: CHAD - Consider perf of multiple calls to VideoClip.
         Used on UI to calc reference time for matching video (../existing-query)
         :return:
         Location of video this match is associated with.
         """
         clip = VideoClip.objects.get(id=self.video_clip_id)
-        return clip.duration * (clip.clip - 1)
+        return "{},{}".format(clip.duration * (clip.clip - 1), clip.duration * clip.clip)
```

Comment:
@fetorres I switched to format to remove the need for a cast via `str` and by way of, it also removes a few operations needed for the `+` operations to work with the immutable object.